.github/workflows/build-release.yml

```diff
@@ -26,7 +26,7 @@ jobs:
       uses: thedoctor0/zip-release@master
       with:
         filename: 'manual-install.zip'
-        exclusions: '*.git* *tests/ *tests/data/ *tests/* *tests/**/*'
+        exclusions: '*.git* *tests/ *tests/data/ *tests/* *tests/**/* *psalm/* *.phpcsignore *.yml *.xml *.xml.dist'
```

Comment:
I'm not sure about not including XMl and yml files, though could be required by packages such as the aws-sdk etc

Reply:
Usually, these are not bundled in the release. I know that in WPCS we add these to `.gitattributes`. These are all development files that are usually not pulled when Composer is used but are present when you clone the repo.

Reply:
Right but that should be dependent on what the deps need. E.g. if we excluded `.json` files that _would_ bread a lot of things, so I don't know for sure that excluding .xml and .yml is a good idea. How do we know that no Yaml / XML hires are needed for all the deps to work correctly?

Reply:
Do you mean, like from vendor files? I'm not sure this exclusion list would look at that (might be wrong tho).

But `.yaml`, `.xml`, `.neon` are usually all config files. I don't think any of the dependencies are using/bundling those.

I can remove the general file extensions, and add the specific files to exclude (config files), shouldn't be a problem.